python/paddle/fluid/tests/unittests/collective/fleet/test_dygraph_recompute_for_eager.py

```diff
@@ -201,7 +244,7 @@ def test_fc_net_with_fp16(self):
     def test_recompute_kwargs(self):
         paddle.set_device("gpu")
         kwargs = {"is_test": False}
-        with self.assertRaises(ValueError):
+        with self.assertRaises(TypeError):
```

Comment:
Why?

Reply:
in the past, kwargs must be only contains  preserve_rng_state,  otherwise, raise ValueError in the past code.
now, kwargs can be passed to function, raise TypeError if as this.
for example:
def fun(a):
    print(a + 10)

fun(9, name="hh")

'TypeError: fun() got an unexpected keyword argument 'name''